src/CMakeLists.txt

```diff
@@ -40,6 +40,18 @@ ADD_LIBRARY(${LIBRARY_NAME}
   command/command.cpp
   )
 
+INSTALL(TARGETS ${PROJECT_NAME}
```

Comment:
This is already defined in the main CMakeLists.txt, but with less options. I think it should be defined only here, but I'm not sure how this works with CMake packages yet.